### PR TITLE
150 discuss remaining notes on vehicle structure

### LINKED ISSUE #203: Add trailer hitch to vehicle structure
**Is your feature request related to a problem? Please describe.**
The position of a trailer hitch might be useful in simulation to attach a trailer to a vehicle.

**Describe the solution you'd like**
Add trailer hitch to the vehicle exterior dynamic group.

@MatthiasThDs will add the definition to the vehicle structure,
@ACMackenthun please create a figure once Matthias has finished the definition

### PR DESCRIPTION
**Pull request by MatthiasThDs**
## Describe your changes

According to https://github.com/asam-ev/OpenMATERIAL/issues/150, I added hitches to the vehicle structure. 

Fixes #203

### CODE REVIEW

**Comment by MatthiasThDs** [comment]
I added hitches to the group_dynamic rather than group_static. Just in case we want to make hitches detachable or rotatable inwards (deactivated state). However, there are so many rotation axes common, that I did not define the hitch origin or rotation axis exactly.

**Comment by ClemensLinnhoff** [comment]
I just did some editorial fixes, like putting the hitch definition in alphabetical order and changing the term "back" to "rear" for harmonization with the other definitions.
I also adjusted the definitions of the origins.

Are these changes okay for you @MatthiasThDs?

If yes, @ACMackenthun can start on the illustrations.

**Comment by MatthiasThDs** [comment]
perfect

File: content/_images/Grp_Hitch_Front.svg
Comment (ACMackenthun) [inline content/_images/Grp_Hitch_Front.svg:1]: I added images illustrating the coordinate systems of the hitches. Please check whether I interpreted the definitions correctly

**Review by ACMackenthun — APPROVED** [review]

File: content/_images/Grp_Hitch_Front.svg
Comment (ClemensLinnhoff) [reply content/_images/Grp_Hitch_Front.svg:1]: Looks good, thanks.

**Pull request merged**